src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="/">
+      <div
+        data-qa="big"
+        class="search-bar search-bar--big"
+      >
+        <button class="vector vector--big"></button>
```

Comment:
Also you don't need this tag, just set background in input tag

Reply:
done